apps/commerce-api/src/main/kotlin/com/loopers/infrastructure/order/ExternalOrderService.kt

```
@@ -0,0 +1,26 @@
+import com.loopers.domain.order.Order
+import com.loopers.support.error.CoreException
+import com.loopers.support.error.ErrorType
+import org.springframework.stereotype.Component
+
+@Component
+class ExternalOrderService {
+
+    fun processOrder(order: Order) {
+        try {
+            sendToExternalSystem(order)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)